api/UIEvent.json

```diff
@@ -231,10 +231,10 @@
               "version_removed": "32"
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "3"
```

Comment:
This is `layerX` and wasn't in https://github.com/WebKit/WebKit/blob/7659cc58c0a8c7c04bc854486cb8355a13c008dd/WebCore/khtml/ecma/kjs_events.cpp#L254, can you research when it was added?

Reply:
Looks like this was added in https://github.com/WebKit/WebKit/commit/e214846b7b5cb8b5b852bb058dd036e9ee35d4fa which predates Safari 1!